tests/unit/src/flixel/FlxObjectTest.hx

```diff
@@ -71,7 +71,65 @@ class FlxObjectTest extends FlxTest
 		object2.velocity.x = -2000;
 		
 		step(60);
-		Assert.isFalse(FlxG.overlap(object1, object2)); 
+		Assert.isFalse(FlxG.overlap(object1, object2));
+	}
+	
+	
+	@Test
+	function testUpdateTouchingFlags():Void
```

Comment:
It seems like there are actually 3 different tests being performed here... It would help readability sense to split them into 3 separate functions I think. That way, you also don't need the comments since the function names can make it clear what's being tested.